src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1199 @@
+            logging.getLogger("aiohttp").setLevel(logging.DEBUG)
+
+        prompted_for_credentials = False
+        username_included_serverpart = False
+
+        if not homeserver:
+            homeserver = facade.input(
+                "Enter Matrix homeserver URL (e.g., https://matrix.org): "
+            )
+            prompted_for_credentials = True
+
+        if not (homeserver.startswith("https://") or homeserver.startswith("http://")):
```

Ruff (0.15.9)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)